pages.es/common/npm-run-script.md

```diff
@@ -0,0 +1,7 @@
+# npm run-script
+
+> Este comando es un alias de `npm run`.
+
+- Muestra la documentación del comando original:
```

Comment:
```suggestion
- Ver documentación para el comando original:
```

Reply:
I have unified the 7 PRs to let the user know what to do. to get documentation provided by tldr. Thank you :)